### DISCUSSION TITLE
UX feature: Scroll to top button at docs

### DISCUSSION THREAD

**Discussion by koupisbean** [filamentphp/filament#3360] [Ideas]
Would appreciate a scroll to top button!

**Comment by danharrin** [comment]
Ctrl/Cmd + Up arrow?

**Reply by koupisbean** [reply]
Yeah, there are ways to do it. But I think it would be nice to have that button there too.

**Reply by danharrin** [reply]
The docs are open source, I would accept a pr - https://github.com/filamentphp/filamentphp.com
